ccd-definition/CaseEventToFields/WorkAllocationIntegrationFields-nonprod.json

```diff
@@ -0,0 +1,150 @@
+[
```

Comment:
maybe rename this to a prod file because it's going into the release branch and since it's a donotshow field, it should be fine?

Reply:
do we need to keep it as nonprod for now for testing purposes alongside HNL LR changes?